test/std-test.cc

```diff
@@ -95,6 +96,23 @@ TEST(std_test, optional) {
 #endif
 }
 
+struct throws_on_move {
+  throws_on_move() = default;
+
+  [[noreturn]] throws_on_move(throws_on_move&&) {
+    throw std::runtime_error("Thrown by throws_on_move");
+  }
+
+  throws_on_move(const throws_on_move&) = default;
+};
+
+template <> struct fmt::formatter<throws_on_move> : formatter<string_view> {
+  auto format(const throws_on_move&, format_context& ctx) const {
```

Comment:
This needs a return type for compatibility with older compilers.

Reply:
My apologies once again. I've fixed this.